### ISSUE TITLE
Not working on BigSur

### ISSUE THREAD

**Issue by castroCrea** [Syed-Umair/electron-window-nspanel#3]
Hello,

I have tried you lib and it work pretty well but since Mac Big Sur, the Panel crash 
![image](https://user-images.githubusercontent.com/20707343/99654227-4f40c480-2a5a-11eb-9be4-a1ef8a30fb21.png)

I have tried to fix it but with no success

Any idea ?

**Comment by Syed-Umair** [comment]
Hi @castroCrea ,
Its crashing because of recent breaking design changes in the MacOS BigSur. I am currently working on a fix for that. Will be rolling out soon. Thank you for using my module.

**Comment by castroCrea** [comment]
Thank you for the reply, I'm also trying if I found something I let you know

**Comment by castroCrea** [comment]
Hello @Syed-Umair,
I hope everything is good on your side?
Do you have any advancement on this ?
Thank you

**Comment by castroCrea** [comment]
@Syed-Umair this comment appear if it can help you to fix your lib https://gist.github.com/ibash/2c9894a8038b71456aeabe413e532c19

**Comment by Syed-Umair** [comment]
> @Syed-Umair this comment appear if it can help you to fix your lib https://gist.github.com/ibash/2c9894a8038b71456aeabe413e532c19

Thanks @castroCrea , i will try that.

**Comment by Syed-Umair** [comment]
Hi @castroCrea ,

Released fix in v1.0.5. Let me know if you face any issues. Thanks for your help.

**Comment by castroCrea** [comment]
I still got the error with multiple window 

![image](https://user-images.githubusercontent.com/20707343/103705797-fd251580-4fab-11eb-8ada-0eef3804f8e3.png)

when i up with react and the other one a simple
```
let win = new PanelWindow({
    width: 710,
    height: 210,
  });
 win.loadURL('https://google.com'
  );

```
